app/models/order.js

```diff
@@ -97,15 +97,15 @@ export default Model.extend({
     }
   ),
 
-  clientIdNumber: Ember.computed("beneficiary", function() {
+  clientIdNumber: Ember.computed("beneficiary.identityNumber", function() {
```

Comment:
those can probably be shortened with `Ember.computed.alias('beneficiary.identityNumber')` :D